### PR TITLE
TD-2804-Cannot-build-CRA-based-apps-with-latest-RUI

### PR DESCRIPTION
**Pull request by stevenmcsorleyipg**
Commented out 4 recent components that were causing build errors.

Closes [TD-2804](https://sce.myjetbrains.com/youtrack/issue/TD-2804/Cannot-build-CRA-based-apps-with-latest-RUI)

## Changes
Fix build errors for CRA-based apps.
This PR addresses the build errors occurring in our CRA based applications when using the latest versions of RUI.
A noted issue was caused by new SVG components added to RUI, specifically the traffic light sign components.

SVG Error Investigation:

While investigating build errors in CRA-based apps, issues with SVG components in RUI were identified:

-- Presence of metadata tags in SVG files.
-- Namespace declarations (xmlns:dc, xmlns:cc, xmlns:rdf, xmlns:svg) not supported by React's JSX.

These elements showed conflicts during the build process. Formatting some of the problematic files allowed the build to progress further but did not fully resolve the issue.

As a temporary solution, we've commented out the exports of the affected SVG components to unblock development and allow updates to the latest RUI version.

## Dependencies
N/A

## UI/UX

The Bug
image

The Fix
image

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
- [ ] Branch has been run in docker.
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [ ] Appropriate tests have been added.
- [x] Lint and test workflows pass.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -27,10 +27,10 @@ export {
...
 export { default as CanvasItem, type CanvasItemProps } from "./CanvasItem";
 export { default as EditLabelDialog } from "./EditLabelDialog";
-export { default as RoadMarking, type RoadMarkingProps } from "./RoadMarking";
-export { default as RoadSurface, type RoadSurfaceProps } from "./RoadSurface";
-export { default as TrafficSign, type TrafficSignProps } from "./TrafficSign";
-export { default as VehiclePath, type VehiclePathProps } from "./VehiclePath";
+// export { default as RoadMarking, type RoadMarkingProps } from "./RoadMarking";
+// export { default as RoadSurface, type RoadSurfaceProps } from "./RoadSurface";
+// export { default as TrafficSign, type TrafficSignProps } from "./TrafficSign";
+// export { default as VehiclePath, type VehiclePathProps } from "./VehiclePath";
```
Comment (mattcorner) [inline src/index.ts:33]: We should also comment out TrafficLight as its part of the same feature set. (Was mentioned in ticket comment)

Comment (mattcorner) [reply src/index.ts:33]: Can we also group the commented exports together at the bottom of the file and leave a comment as to why they are commented out. Leave a link to the bug ticket in YouTrack so can we can trace the detail when we come back and reinstate these in the future.

**Review by mattcorner — CHANGES_REQUESTED** [review]

**Comment by mattcorner** [comment]
@lukemojo This will need a RUI target release and prerelease for testing. Suggest that it just required a patch release unless there is other RUI work in the pipeline. 

Could you also please take @stevenmcsorleyipg through the RUI PR -> prerelease -> release process that you follow when you get a sec?

**Comment by lukemojo** [comment]
> @lukemojo This will need a RUI target release and prerelease for testing. Suggest that it just required a patch release unless there is other RUI work in the pipeline.
> 
> Could you also please take @stevenmcsorleyipg through the RUI PR -> prerelease -> release process that you follow when you get a sec?

No problem @mattcorner. I'll create the pre-release branch shortly and we'll go through the release later.

**Review by mattcorner — APPROVED** [review]
lgtm

**Pull request merged**